src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -25,9 +25,11 @@ def bugzilla_url(email_list = '', desc = '')
     if email_list.length > 1
       cc = ('&cc=' + email_list[1..-1].join('&cc=')) if email_list
     end
-    # rubocop:disable Metrics/LineLength
-    URI.escape("#{@configuration['bugzilla_url']}/enter_bug.cgi?classification=7340&product=openSUSE.org&component=3rd party software&assigned_to=#{assignee}#{cc}&short_desc=#{desc}")
-    # rubocop:enable Metrics/LineLength
+
+    URI.escape(
+      "#{@configuration['bugzilla_url']}/enter_bug.cgi?classification=7340&product=openSUSE.org" \
```

Comment:
@ChrisBr you're right but I need to fix it, since Rubocop complained. I move it to a new commit before I merge.